mbq/metrics/contrib/django/middleware/timing.py

```diff
@@ -0,0 +1,50 @@
+from time import time
+
+from django.conf import settings
+try:
+    from django.utils.deprecation import MiddlewareMixin as MiddlewareDeprecationMixin
+except ImportError:
+    MiddlewareDeprecationMixin = object
+
+import mbq.metrics as metrics
+
+DEFAULT_SETTINGS = {
+    'EXCLUDED_PATHS': []
+}
+
+SETTINGS = DEFAULT_SETTINGS.copy()
+SETTINGS.update(
+    getattr(settings, 'MBQ_METRICS', {})
+)
```

Comment:
1. Is this a pattern you've seen elsewhere? Why do we need a copy of the original settings?
2. Make sure EXCLUDED_PATHS is always a `set` so that `request.path in SETTINGS['EXCLUDED_PATHS']` is `O(1)`

Reply:
1. I think I originally wanted DEFAULT_SETTINGS to be separate but I threw it out. 
2. Done!

Reply:
✅